.github/workflows/scripts/e2e.vsa.gcp.gke.verify.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# With a locally cloned github.com:GoogleCloudPlatform/gke-vsa repository,
+# this script veriries all the VSAs within.
+
+do_verify() {
+    ATTESTATION_FILE="$1"
+    RESOURCE_URI="$2"
+    SUBJECT_DIGEST="$3"
+
+    echo "Verifying $ATTESTATION_FILE"
+
+    slsa-verifier verify-vsa \
+        --attestation-path "$ATTESTATION_FILE" \
+        --resource-uri "$RESOURCE_URI" \
+        --subject-digest "$SUBJECT_DIGEST" \
+        --verifier-id https://bcid.corp.google.com/verifier/bcid_package_enforcer/v0.1 \
+        --public-key-id keystore://76574:prod:vsa_signing_public_key \
+        --public-key-path ./vsa_signing_public_key
+}
+
+iterate() {
+    # find all attestations
+    declare -a FILE_PATHS
+    while IFS= read -r file; do
+        FILE_PATHS+=("$file")
+    done < <(find ./gke-node-images:238739202978 ./gke-master-images:78064567238 -type f)
```

Comment:
Would it be better to search for all intoto.jsonl files, so if they add other top level directories in the future, this script will pick them up?

Reply:
By the time they add support for container images, I suspect they may change the structure of the directories, so I hesitate to simple look for all  intoto.jsonl files.

Reply:
Looks like we fail if we find no files, so SG, this should be sufficient and will catch if they make changes in the future.